vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.h

```diff
@@ -47,6 +47,13 @@ class Promotion {
       ledger::PromotionPtr promotion,
       ledger::ResultCallback callback);
 
+  void SetLastCheckTimerIdForTesting(uint32_t timer_id) {
+      last_check_timer_id_ = timer_id;
+    }
+  void SetRetryTimerIdForTesting(uint32_t timer_id) {
+      retry_timer_id_ = timer_id;
+    }
+
```

Comment:
Would coding standards mean these should be in the .cc rather than inline? They are just for mocking / testing...

Reply:
yup all logic needs to be in cc afaik